Pod/Classes/Logging/Log.swift

```diff
@@ -62,7 +93,15 @@ open class Log {
     ///            your log statements! Use log levels appropriately
     ///            to keep private data out of logs that are sent over
     ///            the Internet.
-    public static var handler: ((Level, String) -> Void)?
```

Comment:
Maybe this should be kept around and marked with a deprecated attribute, and shimmed with overridden `get {} set {}` to the new `globalHandler`

```
@available(*, deprecated, renamed: "globalHandler")
```

Reply:
it's being kept as an Instance level handler. This allow for more selective handling if there are a bunch of different Log instances floating around.